### ISSUE TITLE
[Problem] Image: size of ImagePlane changes when copied in a different document

### ISSUE THREAD

**Issue by alafr** [FreeCAD/FreeCAD#11036]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

When I copy an ImagePlane object from a document to another document, the XSize and YSize of the new copy are different than the original (in the example the copy is 359x466 mm, the original is 133x172 m).
To reproduce:

-   open attached document
-   copy the image plane and the rectangle that has the same size (Ctrl+C)
-   create new document
-   in the new document, paste (Ctrl+V)
-   the image is now smaller, needs a lot of zoom to see it

![image|50x50](https://github.com/FreeCAD/FreeCAD/assets/17966696/e8b1ce92-f9d4-4b72-8776-1124d2a05fbd)
![image](https://github.com/FreeCAD/FreeCAD/assets/17966696/ae791ba4-6826-43da-9157-1c40281ea004)

### Full version info

```shell
OS: Windows 10 build 19044
Word size of FreeCAD: 64-bit
Version: 0.21.1.33694 (Git)
Build type: Release
Branch: (HEAD detached at 0.21.1)
Hash: f6708547a9bb3f71a4aaade12109f511a72c207c
Python 3.10.12, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.5, OCC 7.6.3
Locale: French/France (fr_FR)
Installed mods: 
  * BIM
  * Defeaturing 1.2.1
```

### Subproject(s) affected?

File formats

### Anything else?

[imageplane_test.zip](https://github.com/FreeCAD/FreeCAD/files/12905351/imageplane_test.zip)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wwmayer** [comment]
Confirmed. However, when adding the copy to the source document then the size is correct.

**Comment by alafr** [comment]
It's exact, I scaled the image to match real world dimensions (I don't remember if I used the calibrate button or I manually set the XSize & YSize, but in both cases the behaviour is the same) and the copied image has its size reset to the size it had when it was first inserted.
I also confirm that there is no issue when the copy is in the same document.
The issue happens with Copy / Paste in different documents, drag / drop in different documents, Merge Project command.
There is no issue with Link.
